packages/ui/src/walkthrough-card/WalkthroughCard.css

```diff
@@ -0,0 +1,46 @@
+.ui-WalkthroughCard--CardContainer {
+  display: flex;
+  border-radius: var(--layout-border-radius-xlarge);
+  box-shadow: var(--surface-shadow-none), var(--surface-shadow-none);
+  background-color: var(--surface-color-background-white);
+  max-width: var(--walkthrough-card-max-width);
+  flex-direction: column;
+  font-size: 14px;
+  color: var(--support-border-color-info);
+  padding: var(--layout-padding-large);
+  margin: var(--layout-margin-medium);
```

Comment:
I would assume in our use cases for these I'd want the container to some gap spacing to the cards. This makes a minimum spacing on these that can't be reduced by the parent easily.

Reply:
I'm not seeing this changed. Thoughts on removing it?